pkg/ibmcloudprovider/volume_provider.go

```diff
@@ -60,8 +61,9 @@ func NewIBMCloudStorageProvider(configPath string, logger *zap.Logger) (*IBMClou
 		conf.VPC.APIVersion = "2021-04-20" // setting default values
 	}
 
+	var clusterInfo utilsConfig.ClusterConfig
 	logger.Info("Fetching clusterInfo")
-	clusterInfo, err := utils.NewClusterInfo(logger)
+	clusterInfo, err = utilsConfig.GetClusterInfo(*k8sClient, logger)
```

Comment:
@prankulmahajan Please add this logic or If not I will take care of this as part of ENI 

https://github.com/IBM/ibm-csi-common/blob/master/pkg/ibmcloudprovider/volume_provider.go#:~:text=%7D-,var%20clusterInfo%20utilsConfig.ClusterConfig,%7D,-//Initialize%20the%20clusterVolumeLabel

Reply:
This can be part of ENI PR only